src/aes/Makefile.in

```diff
@@ -11,7 +11,7 @@ YASM = @YASM@
 AR = @AR@
 FIND = @FIND@
 RM = /bin/rm -f
-GCCV44 := $(shell expr `$(CC) -dumpversion` \>= 4.4)
+GCCV44 := $(shell echo `$(CC) -dumpversion` \>= 4.4 | bc)
```

Comment:
It will fail to clang.

Reply:
On clang, this ends up checking the clang version, right?

Reply:
There are many dots